script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -68,7 +68,7 @@ With a bow, Hisui enters the room.
 [sha:311c08f94f0b45acf899feeffbe2a7b0eade89a4]{
 -- Page 6, Offset 16578.
 -- ……琥珀さんにはちょっと熱があるから、量は少なめだって伝えてほしい」
-...Can you let Kohaku-san know that since I have a slight fever, I'd like a smaller portion today?"
+...Can you let Kohaku-san know since I have a slight fever, I'd like a smaller portion today?"
```

Comment:
original in this case is better imo

Reply:
Irish just running replace-alls on `s/that//g`

Reply:
lmao then I'll gradually keep more of them, no need to be so allergic